wgsl/index.bs

```diff
@@ -9076,9 +9097,16 @@ from their respective desugaring translations to [[#loop-statement|loop]] statem
 
 In [[#switch-statement|switch]], a [=default-alone clause=] block is treated exactly like a [=case clause=] with regards to uniformity.
 
-Note: If the set of behaviors (see [[#behaviors]]) for an if, switch, or loop statement is {Next}, this means that we either did not diverge within the statement, or we reconverged, so we pick the node corresponding to control flow at the start of the statement as the node corresponding to control flow at the exit of the statement.
+GPUs are incentivized to minimize the amount of [=uniform control
+flow|non-uniform control flow=]; however, the points at which invocations can
+be said to be uniform varies depending on a number of factors.
+WGSL's static analysis conservatively assumes a return to uniform control flow
+occuring at the end of if, switch and loop statements if the
+[[#behaviors|behavior]] for the statement is {Next}.
+This is modelled in the preceding table as the resulting control flow node
```

Comment:
Minor comment: should `modeled` have the preference? The rest of the spec tends towards such spelling, including `behavior` in this PR.

Reply:
Yeah, "modeling" and "traveling" are similar cases.  I tend to use single "l".

Reply:
I unified all behavior while I was at it: behaviour -> behavior.

Reply:
Thank you!